### PR TITLE
Guide layout

### PR DESCRIPTION
**Pull request by djalil-bouhachi**
Guide-layout

### CODE REVIEW

**Comment by BaardBouvet** [comment]
Perhaps we should add some time estimates for the tutorials? 30 minutes, etc.

File: Guide-template.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
+:orphan:
+
+Guide title
+===========
+
+This is an introduction to this guide. Lorem ipsum dolor coloris mati tamores dow.
+
+.. admonition:: Pre-requists
+
+   * Get familar with concepts
```
Comment (BaardBouvet) [inline Guide-template.rst:10]: These should be links

**Review by BaardBouvet — CHANGES_REQUESTED** [review]

**Comment by BaardBouvet** [comment]
I think this looks great, see my comments on the review!

File: Guide-template.rst (hunk 1 above)
Comment (djalil-bouhachi) [reply Guide-template.rst]: OK

**Comment by djalil-bouhachi** [comment]
@BaardBouvet Added 2 ways to show reading time estimates. 
Guide layout

**Comment by djalil-bouhachi** [comment]
Added 2 ways to show estimated reading time @BaardBouvet 
Guide layout

**Pull request closed without merging**